pkg/kopia/command/parse_command_output.go

```diff
@@ -327,6 +331,93 @@ func parseKopiaProgressLine(line string, matchOnlyFinished bool) (stats *Snapsho
 	}
 }
 
+// SnapshotRestoreStats is a container for stats parsed from the output of a
+// `kopia snapshot restore` command.
+type SnapshotRestoreStats struct {
+	FilesProcessed  int64
+	SizeProcessedB  int64
+	FilesTotal      int64
+	SizeTotalB      int64
+	ProgressPercent int64
+}
+
+// SnapshotStatsFromSnapshotRestore parses the output of a `kopia snapshot
+// restore` execution for a log of the stats for that execution.
+func SnapshotStatsFromSnapshotRestore(snapRestoreStderrOutput string) (stats *SnapshotRestoreStats) {
```

Comment:
```suggestion
func RestoreStatsFromRestoreOP(snapRestoreStderrOutput string) (stats *SnapshotRestoreStats) {
```
maybe this or similar?

Reply:
I've renamed it to `RestoreStatsFromRestoreOutput` to make it as straightforward as possible.